pkg/serverless/daemon/daemon.go

```diff
@@ -342,6 +340,14 @@ func (d *Daemon) ComputeGlobalTags(configTags []string) {
 	}
 }
 
+func (d *Daemon) setTraceTags(tagMap map[string]string) bool {
+	if d.TraceAgent != nil && d.TraceAgent.Get() != nil {
+		d.TraceAgent.Get().SetGlobalTagsUnsafe(tags.BuildTracerTags(tagMap))
+		return true
+	}
+	return false
```

Comment:
Do you want to do something with this return value? This could be an `error`, or you could just return nothing if `setTraceTags` does the right thing no matter what and it doesn't matter what the outcome is.